### ISSUE TITLE
Bookcast app fails in build

### ISSUE THREAD

**Issue by ebruchez** [orbeon/orbeon-forms#3045]
Navigate to `/xforms-bookcast/`, getting an error.

**Comment by ebruchez** [comment]
Worked with 2016.2. What changed? Getting:

```
Exception: org.orbeon.oxf.processor.validation.SchemaValidationException                                              |
|----------------------------------------------------------------------------------------------------------------------|
|orbeon.oxf.xforms.schema.MSVGrammarReaderController|error                         |MSVGrammarReaderController.sca|  72|
|org.orbeon.msv.reader.Controller                   |error                         |Controller.java               |  49|
|org.orbeon.msv.reader.GrammarReader                |reportError                   |GrammarReader.java            | 746|
|org.orbeon.msv.reader.GrammarReader                |reportError                   |GrammarReader.java            | 704|
|org.orbeon.msv.reader.xmlschema.ElementDeclState$1 |patch                         |ElementDeclState.java         | 104|
|org.orbeon.msv.reader.GrammarReader                |runBackPatchJob               |GrammarReader.java            | 689|
|org.orbeon.msv.reader.GrammarReader                |runBackPatchJob               |GrammarReader.java            | 677|
|org.orbeon.msv.reader.xmlschema.XMLSchemaReader    |wrapUp                        |XMLSchemaReader.java          | 746|
|org.orbeon.msv.reader.xmlschema.RootState          |endSelf                       |RootState.java                |  28|
|org.orbeon.msv.reader.SimpleState                  |endDocument                   |SimpleState.java              | 111|
|org.xml.sax.helpers.XMLFilterImpl                  |endDocument                   |XMLFilterImpl.java            | 497|
|org.orbeon.saxon.event.ContentHandlerProxy         |close                         |ContentHandlerProxy.java      | 274|
|org.orbeon.saxon.event.ProxyReceiver               |close                         |ProxyReceiver.java            |  90|
|org.orbeon.saxon.event.ComplexContentOutputter     |close                         |ComplexContentOutputter.java  | 507|
|org.orbeon.saxon.event.ReceivingContentHandler     |endDocument                   |ReceivingContentHandler.java  | 219|
|org.orbeon.dom.io.SAXWriter                        |endDocument                   |SAXWriter.scala               | 153|
|org.orbeon.dom.io.SAXWriter                        |writeDocument                 |SAXWriter.scala               |  68|
|org.orbeon.dom.io.SAXWriter                        |parse                         |SAXWriter.scala               | 127|
|org.orbeon.saxon.event.Sender                      |sendSAXSource                 |Sender.java                   | 404|
|org.orbeon.saxon.event.Sender                      |send                          |Sender.java                   | 193|
|org.orbeon.oxf.xml.IdentityTransformerWithFixup    |transform                     |IdentityTransformerWithFixup.s|  31|
|org.orbeon.oxf.xml.TransformerWrapper              |transform                     |TransformerUtils.java         | 714|
|org.orbeon.oxf.xml.TransformerUtils                |sourceToSAX                   |TransformerUtils.java         | 543|
|org.orbeon.oxf.xml.TransformerUtils                |writeDom4j                    |TransformerUtils.java         | 564|
|org.orbeon.oxf.xforms.XFormsModelSchemaValidator   |loadInlineGrammar             |XFormsModelSchemaValidator.jav| 618|
```

**Comment by ebruchez** [comment]
Works if adding `xmlns=""` on `xs:schema`. Suspecting this followed the fix for #339.

**Comment by ebruchez** [comment]
I am not 100% sure but I think that adding `xmlns=""` is correct. So doing that.
